.github/workflows/actions_build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-12, windows-latest]
-        java: [17]
+        java: [17, 21]
```

Comment:
Should we upgrade `BUILD_JDK_VERSION` to 21 as well?

Reply:
Yes, I updated